insights/parsers/tests/test_ifcfg.py

```diff
@@ -147,6 +156,10 @@
 
 IFCFG_PATH_NAMED_BOND_MASTER = "etc/sysconfig/network-scripts/ifcfg-eth0"
 
+IFCFG_PATH_NAMED_TEAM_MASTER = "etc/sysconfig/network-scripts/ifcfg-eth0"
```

Comment:
You need to remove the `IFCFG_PATH_NAMED_BOND_MASTER` and `IFCFG_PATH_NAMED_BOND_MASTER`

Reply:
@xiangce removed the `IFCFG_PATH_NAMED_TEAM_MASTER` and `IFCFG_PATH_NAMED_BOND_MASTER` and updated the test case for `IFCFG_PATH_ETH1` and `IFCFG_PATH_ETH2` path which are used in test case